CONTRIBUTING.md

```diff
@@ -316,6 +316,7 @@ More specifically an error should
 - implement Display using `write_err!()` macro if a variant contains an inner error source.
 - have `Error` suffix
 - call `internals::impl_from_infallible!
+- not have `Error` suffix on enum variants
```

Comment:
Maybe say that the Error is an `enum`?
Is that the assumption? I can't discern.
Also it is somewhat confusing, check 2 lines above for "have `Error` suffix"

Maybe something like this?

```suggestion
- not have `Error` suffix on its variants
```

But then this would imply that `Error`s are always `enum`s.
As a counter-argument, [`std::io::Error`](https://doc.rust-lang.org/std/io/struct.Error.html)
is a struct.

Reply:
I think we should:

* Move the two lines next to each other so it's clear they are not a mistake/conflict
* Change the first line to "have `Error` suffix on error types"

I think the existing text "not have `Error` suffix on enum variants" is clear. It says that *if* your error type is an enum, then its variants shouldn't have a suffix.

Reply:
Nice, done. Also includes slightly unrelated addition of a full stop.